### ISSUE TITLE
Initial convnet not learning anything

### ISSUE THREAD

**Issue by jspunda** [jspunda/prostatex#25]
Just pushed new code to keras_net/train.py which contains a simple convnet (conv32, conv32, maxpool, conv64, conv64, maxpool, dense512). Good news is that it at least runs the net for our loaded lesions and the labels, but it doesn't seem to be learning anything. The predictions before and after training are pretty much the same. After the first epoch the validation accuracy doesn't change anymore. For each of the lesions it has a very high probability for one class and almost zero for the other. 

I've already tried different optimizers, like SGD and Adam, but right now it's using rmsprop and I've also tweaked the learning rate ranging from 0.01 to 0.00001, but no real change. Not sure what's causing this. I would have expected to see at least some learning with a network like this.

Just for testing/debugging purposes I tried to train a network with just two layers and only a few neurons in it. This produces the same type of output as the convnet; everything classified as the same class.

Currently it's splitting the data in a 70-30 split using train_test_split from scikit learn.  Perhaps it generates a very unbalanced split?

Maybe someone else has an idea.
